boms/preview-ee/pom.xml

```diff
@@ -50,6 +50,7 @@
         <version.jakarta.validation.jakarta-validation-api>3.1.0</version.jakarta.validation.jakarta-validation-api>
         <version.jakarta.websocket.jakarta-websocket-api>2.2.0</version.jakarta.websocket.jakarta-websocket-api>
         <version.jakarta.ws.rs.jakarta-ws-rs-api>4.0.0</version.jakarta.ws.rs.jakarta-ws-rs-api>
+        <version.org.glassfish.expressly>6.0.0-M1</version.org.glassfish.expressly>
```

Comment:
I just noticed that GlassFish 8 is using Glassfish Expressly 6.0.0-M2 https://github.com/eclipse-ee4j/glassfish/blob/8.0/nucleus/parent/pom.xml#L74 6.0.0-M2 which includes exactly one fix https://github.com/eclipse-ee4j/expressly/pull/14

Reply:
@bstansberry should we switch to M2 since it is newer?

Reply:
As I expected, build fails if I try to use M2:

```
Downloading from jboss-public-repository-group: https://repository.jboss.org/nexus/content/groups/public/org/glassfish/expressly/expressly/6.0.0-M2/expressly-6.0.0-M2.pom
Downloading from central: https://repo.maven.apache.org/maven2/org/glassfish/expressly/expressly/6.0.0-M2/expressly-6.0.0-M2.pom
[WARNING] The POM for org.glassfish.expressly:expressly:jar:6.0.0-M2 is missing, no dependency information available
Downloading from jboss-public-repository-group: https://repository.jboss.org/nexus/content/groups/public/org/glassfish/expressly/expressly/6.0.0-M2/expressly-6.0.0-M2.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/glassfish/expressly/expressly/6.0.0-M2/expressly-6.0.0-M2.jar

...

[ERROR] Failed to execute goal on project wildfly-preview-feature-pack: Could not resolve dependencies for project org.wildfly:wildfly-preview-feature-pack:pom:34.0.0.Beta1-SNAPSHOT: The following artifacts could not be resolved: org.glassfish.expressly:expressly:jar:6.0.0-M2 (absent): Could not find artifact org.glassfish.expressly:expressly:jar:6.0.0-M2 in jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/) -> [Help 1]
```

Reply:
Thanks for trying!